src/app/about-modal/about-modal.module.ts

```diff
@@ -1,11 +1,11 @@
 import { NgModule }         from '@angular/core';
 import { CommonModule }     from '@angular/common';
 
-import { ModalModule } from 'ngx-modal';
+import { ModalModule } from 'ngx-bootstrap';
```

Comment:
This should be ng2 not ngx, we have not upgraded to ngx-bootstrap yet due to bugs in it.